### ISSUE TITLE
 [BUG] Variables don't work in MULTI PART Body, values not extracted

### ISSUE THREAD

**Issue by Gabrielcefetzada** [usebruno/bruno#1236]
First of all, i already found and read this. [issue](https://github.com/usebruno/bruno/issues/233)
It's a similar problem, but my body is not json. 

My variables are correctly been set by one post request script at "search request"
```js
let body = res.getBody();
bru.setVar("hotelCacheId",body.data.cacheId);
```
![image](https://github.com/usebruno/bruno/assets/63877012/bd75fd29-f4b6-476d-ae15-b464b8fec1fb).

But, when i switch to another requests, that needs this `hotelCacheId` variable, i got the following: {{hotelCacheId}}
Not the true hash value showed above. 
![image](https://github.com/usebruno/bruno/assets/63877012/323f4713-f229-4b89-9fa5-2fb52e9768b6)

 OS: Ubuntu 22
 Bruno Version: v1.5.0

**Comment by Its-treason** [comment]
What kind of body are you using in the second screenshot?

**Comment by Gabrielcefetzada** [comment]
> What kind of body are you using in the second screenshot?

Multi Part

**Comment by Its-treason** [comment]
Yeah, that's a problem with that specific body type, see #352. I already have a PR for this open here #1217 that needs to be reviewed and merged, this is hopefully fixed with one of the next releases.
